PayGreen/paygreen-ui
# PGF-114 - Auto-deploy Storybook

## How to test ?
- After merge
- Try push a new tag on a branch
- Then check branch gh_pages, docs files must be updated and GitHub Pages too

_Reload your browser if GitHub Pages doesn't update._
